bundles/AdminBundle/Controller/Admin/DataObject/ClassController.php

```diff
@@ -217,6 +217,10 @@ public function getAction(Request $request)
     {
         $class = DataObject\ClassDefinition::getById($request->get('id'));
         $class->setFieldDefinitions([]);
+        $isWriteable = $class->isWritable();
+        $class = json_encode($class);
```

Comment:
Why not `$class->getObjectVars()` or `object2array()` or all together `\object2array($class->getObjectVars())`?